assert/assert.go

```diff
@@ -90,6 +90,14 @@ func (assert Assert) EqFloat(actual, expected float64) {
 	}
 }
 
+// EqFloat fails the test if two floats aren't equal. NaNs are considered equal.
+func (assert Assert) EqApproximate(actual, expected, epsilon float64) {
+	delta := actual - expected
+	if !(-epsilon < delta && delta < epsilon) {
```

Comment:
should feel like they should always differ, since ... what does it mean to be epsilon away from these values anyway?